algorithms/benchmark/neighbors_search_algorithms_test.py

```diff
@@ -0,0 +1,116 @@
+import unittest
+
+
+from bruteforce import bruteforce, optimized_bruteforce
+
+
+class TestBenchmark2D(unittest.TestCase):
+
+    def setUp(self):
+        self.search_radius = 0.2
+        self.epsilon = 10e-8
+        self.square_len = 1.0
+        self.all_points = [
```

Comment:
Please combine `all_points` and `answers` into 1 list so it will be possible to see points and expected neighbours without scrolling up and down.